src/components/Pagination/Pagination.tsx

```diff
@@ -388,23 +392,25 @@ export const Pagination: FC<PaginationProps> = React.forwardRef(
                       }
                     />
                   ) : (
-                    <Pager
-                      currentPage={_currentPage}
-                      key="pager"
-                      locale={locale}
-                      onCurrentChange={handleCurrentChange}
-                      pageCount={getPageCount()}
-                      quickNextIconButtonAriaLabel={
-                        quickNextIconButtonAriaLabel
-                      }
-                      quickPreviousIconButtonAriaLabel={
-                        quickPreviousIconButtonAriaLabel
-                      }
-                      simplified={true}
-                      showLast={
-                        !layout.includes(PaginationLayoutOptions.NoLast)
-                      }
-                    />
+                    moreThanOnePage && (
```

Comment:
We may wish to make this opt-in for now as the `Pagination` component has been displaying when only 1 page for quite a few cycles now. Perhaps with a new boolean prop like `hideWhenSinglePage`? Then default to `false` until we might coordinate the change?

Reply:
Agree, added `hideWhenSinglePage` prop as suggested

Reply:
@rnori-eightfold You should update `Pager` to display the `pageCount` when `!hideWhenSinglePage`, currently its compare only is relevant to `!simplified` scenarios. Perhaps keep the same condition and breakout `simplified` to its own conditional `<li>` using `simplified && pageCount > 0 && showLast` as the condition? 

https://github.com/rnori-eightfold/octuple/blob/rnori-eightfold/simplified-pager-bug-fixes/src/components/Pagination/Pager.tsx#L230

Reply:
@dkilgore-eightfold Thanks for pointing out. Pushed a [fix](https://github.com/EightfoldAI/octuple/pull/581/commits/1b5d2f25a46f7f5cb0c83ceb9d0a6a1a4eb34306) for this